alloccli/alloc_output_handler.py

```diff
@@ -194,9 +194,9 @@ def print_table(self, alloc, entity, rows, only_these_fields, sort=False, transf
             # FIXME: This shows the header for the CSV. It should have a CLI
             # option to turn it on, but for now, just print it always and let
             # the user filter it out if necessary.
-            csv_table.writerow([unicode(s).encode('utf-8') for s in field_names])
+            csv_table.writerow([str(s).encode('utf-8') for s in field_names])
             for row in rows:
-                csv_table.writerow([unicode(s).encode('utf-8') for s in row])
+                csv_table.writerow([str(s).encode('utf-8') for s in row])
```

Comment:
I thought all strings were utf8 by default in python3... maybe I'm wrong.

Reply:
you are right, I've fixed it in a commit :)